blog/vr-volume-viewer.md

```diff
@@ -11,24 +15,27 @@ tags:
 
 ---
 
-# Building tools for VR scientific visualization – VR Volume viewer
+<center># Building tools for VR scientific visualization – VR Volume viewer</center>
```

Comment:
<h1 style="text-align: center;">Building tools for VR scientific visualization – VR Volume viewer</h1>

Reply:
uses h1 tags, text-align=center